src/app/find/errorReports/ErrorReportInformationSelect.jsx

```diff
@@ -49,19 +49,17 @@ const ErrorReportInformationSelect = (props) => {
       </div>
 
       <div className="mx-4 mb-1">
-        <Link to={`${props.match.url}/text`}>
-          <Button
-            primary
-            fluid
-            onClick={() => {} /* URL navigation handled by Link component wrapper */}
-          >
-            {
-              (!generalLocationError && !errorReportServices.length)
-                ? 'Skip'
-                : 'Next'
-            }
-          </Button>
-        </Link>
+        <Button
+          primary
+          fluid
+          onClick={goToErrorReportTextScreen}
```

Comment:
I'd rename this to `onSubmit` too, as it''s named in the other page.
Since we're making these components purely presentational, they don't need to know what happens when they're finished - that's up to the container, and they just need to call the function it gave them.